ReferencePluginG/ControlG.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+using Paratext.PluginInterfaces;
+
+namespace ReferencePluginG
+{
+	public partial class ControlG : EmbeddedPluginControl
+	{
+		Thread m_UpdateThread;
+
+		// m_Exists is used for two purposes:
+		// 1. So we don't create a second window if Paratext closed with one open.
+		// 2. So we don't update the text if the window is closing.
+		public static bool m_Exists;
```

Comment:
Static members should have a s_ prefix instead of m_ prefix. (Note that this is not a convention followed by Paratext code, but the demo plugins use this way of naming variables.) Also, FWIW, the first letter following the underscore is normally lowercase. (I don't know that we especially care about these details in the demo or other reference plugins, but if you're into mindless consistency, that's the guideline.)